### PR TITLE
Images, Search, Home Page and Notifications fix

### PR DESCRIPTION
**Pull request by nar1n**

### CODE REVIEW

File: src/MangaPill/MangaPill.ts [hunk 1]
```diff
@@ -113,7 +113,10 @@ export class MangaPill extends Source {
 
     async searchRequest(query: SearchRequest, metadata: any): Promise<PagedResults> {
         let page: number = metadata?.page ?? 1
-        let genres = '&genre=' + (query.includeGenre ?? []).join('&genre=')
+        let genres: string = (query.includeGenre ?? []).join('&genre=')
```
Comment (ConradWeiser) [inline src/MangaPill/MangaPill.ts:116]: Currently the searching system doesn't handle more than the title. The other options such as `includeGenre` were an old prototype.

I believe it's been determined that the advanced search system in the future will be a completely independent function. As such, you don't need to support things such as `includeGenre` whatsoever. If it simplifies this source a little.

File: src/MangaPill/MangaPill.ts [hunk 2]
```diff
@@ -129,7 +132,7 @@ export class MangaPill extends Source {
         let request = createRequestObject({
             url: `${MANGAPILL_DOMAIN}/search`,
             method: "GET",
-            param: `?page=${page}&title=${encodeURIComponent(query.title ?? '')}${format}${status}${genres}`
+            param: `?q=${encodeURIComponent(query.title ?? '')}${format}${status}${genres}&page=${page}`
```
Comment (ConradWeiser) [inline src/MangaPill/MangaPill.ts:135]: See the comment above, this could be cleaned down to just contain page information and the query title

**Review by ConradWeiser — APPROVED** [review]
After another review later, the above comments really aren't required to be handled. While it doesn't do anything at the moment, having that as a reference for whenever advanced search is implemented would be useful.

I've loaded this into a testing repository and played around with it for a while on my device. It seems to work fine for me!

Thank you for your work!

**Review by Aur0raN — APPROVED** [review]
Tested it and its good to go!

**Pull request merged**